# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1295513436 18000
#      Thu Jan 20 03:50:36 2011 -0500
# Node ID 5c680b69b4e2ec2907d19ae5ab114ef62f55136f
# Parent  06cfda4c07e5419e7908e7442fdac3d220b3c4f4
bootstrap: update from gnulib sources

diff --git a/ChangeLog b/ChangeLog
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,8 +1,12 @@
+2011-01-20  John W. Eaton  <jwe@octave.org>
+
+	* bootstrap: Update from gnulib sources.
+
 2011-01-20  John W. Eaton  <jwe@octave.org>
 
 	* configure.ac, NEWS: Update for backend -> graphics_toolkit change.
 
 2010-01-17  Rik  <octave@nomad.inbox5.com>
 
 	* NEWS: Add quadcc to list of new functions added.
 
diff --git a/bootstrap b/bootstrap
--- a/bootstrap
+++ b/bootstrap
@@ -1,11 +1,11 @@
 #! /bin/sh
 # Print a version string.
-scriptversion=2010-11-20.03; # UTC
+scriptversion=2011-01-14.08; # UTC
 
 # Bootstrap this package from checked-out sources.
 
 # Copyright (C) 2003-2011 Free Software Foundation, Inc.
 
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by
 # the Free Software Foundation, either version 3 of the License, or
@@ -457,17 +457,17 @@ cleanup_gnulib() {
   exit $status
 }
 
 git_modules_config () {
   test -f .gitmodules && git config --file .gitmodules "$@"
 }
 
 gnulib_path=`git_modules_config submodule.gnulib.path`
-: ${gnulib_path:=gnulib}
+test -z "$gnulib_path" && gnulib_path=gnulib
 
 # Get gnulib files.
 
 case ${GNULIB_SRCDIR--} in
 -)
   if git_modules_config submodule.gnulib.url >/dev/null; then
     echo "$0: getting gnulib files..."
     git submodule init || exit $?
